specification/columns/commitmentdiscountconsumedquantity.md

```diff
@@ -0,0 +1,38 @@
+# Commitment Discount Consumed Quantity
+
+Commitment Discount Consumed Quantity relates to [*commitment-based discounts*](#glossary:commitment-based-discount), not to be confused with [*negotiated discounts*](#glossary:negotiated-discount), and is the numeric value of [CommitmentDiscountUnits](#commitmentdiscountunit) that is either applied to a metered [*resource*](#glossary:resource) or [*service*](#glossary:service) or is unused over a [*charge period*](#glossary:chargeperiod).
+
+The CommitmentDiscountConsumedQuantity column adheres to the following requirements:
+
+* CommitmentDiscountConsumedQuantity MUST be present in the billing data when the provider supports *commitment-based discounts*.
+* CommitmentDiscountConsumedQuantity MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements.
+* CommitmentDiscountConsumedQuantity MUST be the positive, used quantity of a corresponding *commitment-based discount* by a metered *resource* or *service* over the charge period when [ChargeCategory](#chargecategory) is "Usage", [CommitmentDiscountStatus](#commitmentdiscountstatus) is "Used", and [ChargeClass](#chargeclass) is not "Correction".
+* CommitmentDiscountConsumedQuantity MUST be the remaining, positive, unused quantity for the corresponding *commitment-based-discount* over the charge period when *ChargeCategory* is "Usage", *CommitmentDiscountStatus* is "Unused", and *ChargeClass is not "Correction".
+* CommitmentDiscountConsumedQuantity MAY be negative or null if *ChargeClass* is "Correction".
```

Comment:
I don't know the inter-workings around a return but I'd guess that since the return window is 7 days and billing period's close 2-3 weeks into the next month, this would *not* be a correction.  That said, it probably makes sense to change to `negative, 0, or null` too.